dev/src/serializer.ts

```diff
@@ -53,12 +52,15 @@ export interface Serializable {
  */
 export class Serializer {
   private createReference: (path: string) => DocumentReference;
+  private createInteger: (n: number | string) => number | BigInt;
 
   constructor(firestore: Firestore) {
     // Instead of storing the `firestore` object, we store just a reference to
     // its `.doc()` method. This avoid a circular reference, which breaks
     // JSON.stringify().
     this.createReference = path => firestore.doc(path);
+    this.createInteger = n =>
```

Comment:
Should we validate the string so that `Number(n)` won't break on bad inputs?

Reply:
These values come from the backend and IIRC any issues with it so far (we already used this code below). Opting not do this for now.